gradle/libs.versions.toml

```diff
@@ -3,8 +3,8 @@ h2 = "2.2.224"
 junit = "5.10.3"
 kotlin = "2.0.0"
 kotlin-browser = "1.0.0-pre.764"
-kotlinx-coroutines-core = "1.9.0-RC"
-kotlinx-coroutines-test = "1.8.1"
+kotlinx-coroutines-core = "1.8.0"
```

Comment:
Unify those two versions into one `kotlinx-coroutines`. Also, double check which version the platform uses.

Reply:
they were different, was a refuse? (1.9 and 1.81)